lib/feedsme/index.js

```diff
@@ -438,11 +438,11 @@ Feedsme.prototype._triggerStrategy = function _triggerStrategy({ registryRootPkg
  * @returns {Promise} to be resolved
  * @api private
  */
-Feedsme.prototype._expandVersion = async function ({ version, pkg, env }) {
+Feedsme.prototype._expandVersion = async function ({ Version, version, pkg, env }) {
   const npm = this.config.get('npm');
 
   this.log.info(`Getting attachments for ${ pkg.name }`);
-  const body = await thenify(version, 'getAttachment', npm);
+  const body = await Version.getAttachment({ pkg: version, read: npm });
```

Comment:
you can just grab `Version` from `this.models` in this function above rather than passing it as an argument